llvm/test/CodeGen/DirectX/any.ll

```diff
@@ -0,0 +1,105 @@
+; RUN: opt -S -dxil-op-lower < %s | FileCheck %s
+
+; Make sure dxil operation function calls for any are generated for float and half.
+
+; CHECK:icmp ne i1 %{{.*}}, false
```

Comment:
Folks usually put a space between `CHECK:` and the thing being checked (`CHECK: icmp ...`). I find that slightly more readable.

Reply:
That was my thinking as well, but all the other tests in this directory lacked the space and I didn't want to go against the the grain.